apps/namada-interface/src/App/AccountOverview/AccountOverview.tsx

```diff
@@ -27,11 +27,34 @@ import {
 } from "./AccountOverview.components";
 import { DerivedAccounts } from "./DerivedAccounts";
 
+import { atom, useAtomValue, useSetAtom } from "jotai";
+import { loadable } from "jotai/utils";
+import { accountsAtom, balancesAtom } from "slices/accounts";
+import { balanceToFiatAtom } from "slices/coins";
+import { fiatCurrencyAtom } from "slices/settings";
+
 //TODO: move to utils when we have one
 const isEmptyObject = (object: Record<string, unknown>): boolean => {
   return object ? Object.keys(object).length === 0 : true;
 };
 
+const totalBalanceAtom = atom(async (get) => {
+  const accounts = await get(accountsAtom);
+  const balances = get(balancesAtom);
+  const balanceToFiat = await get(balanceToFiatAtom);
+  const fiatCurrency = get(fiatCurrencyAtom);
+
+  return accounts.reduce((acc, account) => {
+    const balance = balances[account.address];
+
+    if (balance === undefined) {
```

Comment:
I was always checking with the `typeof`, not exactly sure why. So it turns out the `typeof` check is a bit safer in some edge cases, but `=== undefined` is faster. Anyways, it does not matter in this case and also typescript mitigates some of those :) There are nice answers in the [stackoverflow](https://stackoverflow.com/questions/4725603/variable-undefined-vs-typeof-variable-undefined)